photoprism.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "PhotoPrism-Test",
+    "release": "12.4-RELEASE",
```

Comment:
Let's please use 13.1-RELEASE ?

Reply:
I'm afraid I can't.

At this point there is no pre-built binary for 13.1-RELEASE. Only for 12: https://github.com/psa/photoprism-freebsd-port/releases/tag/2022-11-18.